examples/guestbook-go/redis-master-service.json

```diff
@@ -12,7 +12,8 @@
       "ports": [
         {
           "port":6379,
-          "targetPort":"redis-server"
+          "targetPort":"redis-server",
+          "nodePort":0
```

Comment:
> e2e test uses this json file to create services, and I changed it because the validator complained about the schema error (nodePort is required).

That strikes me as odd.  I thought only certain `ServiceTypes` required `nodePort`.